subjects/java/checkpoints/date-formatter/README.md

```diff
@@ -0,0 +1,70 @@
+## Date Formatter
+
+### Instructions
+
+Write a class `DateFormatter` that has three private attributes:
+
+- `date`: the date to be formatted (as a UNIX time in seconds).
+- `format`: the format to convert the date to.
+- `formattedDate`: contains the date converted into the given format.
+
+Create getters for all the attributes and setters for `date` and `format`. The conversion should happen when you set the `format` or the `date`. If the `format` is not correct, the conversion shouldn't happen. Additionally, create methods to output the date in the specified `format`, and constructors with only the `date`, with `date` and `format`, and a default constructor.
+
+The default date is the current date, and the format as follows `DD/MM/YYYY`, The format is case insesitive
```

Comment:
There are two small typos:
1. the `,` should be `.` - or `The` should not be capitalized
2. `insensitive` is misspelt.